src/fides/api/util/tcf/tcf_experience_contents.py

```diff
@@ -398,6 +405,11 @@ def build_purpose_or_feature_section_and_update_vendor_map(
                     has_vendor_id=bool(
                         vendor_id
                     ),  # Has_vendor_id will let us separate data between systems and vendors
+                    cookie_max_age_seconds=privacy_declaration_row.system_cookie_max_age_seconds,
+                    uses_cookies=privacy_declaration_row.system_uses_cookies,
+                    cookie_refresh=privacy_declaration_row.system_cookie_refresh,
+                    uses_non_cookie_access=privacy_declaration_row.system_uses_non_cookie_access,
+                    legitimate_interest_disclosure_url=privacy_declaration_row.system_legitimate_interest_disclosure_url,
```

Comment:
This makes sense given how we're currently structuring the TCF overlay contents, but worth noting this creates duplication when we intentionally created a separate "vendor relationships" section to reduce duplication in the first place.  I think your implementation is fine, provided we're not making it harder for the FE to combine vendors.


---
More info below:

The TCF overlay has three vendor sections: 1) Vendor Consent 2) Vendor Legitimate Interests and 3) Vendor Relationships.  The same vendor can show up in one or many sections, provided it has associated content for those sections.  For example, if a vendor has a purpose it's collecting with a consent legal basis, it will show up in 1) Vendor Consent section.  If a Vendor has a Special Feature (plus other criteria) it will show up in 3) Vendor Relationships.    

3) Vendor Relationships surfaces Vendors with Special Purposes, Features, and Special Features to reduce duplicating this across 1) and 2).  Now we're duplicating Cookie info across 1) 2) and 3).  

Other alternatives might be to have the Vendor always show up in 3) if it appears in 1) or 2) and only collect cookie information there.

Reply:
hmmm, i still don't totally understand this even after reading it a few times, and i know you'd also tried to point me to it last week! 😆 

i feel like we should probably defer to what's best for the FE as long as it's reasonable, and let's just get it in place here while we're at it... @allisonking, thoughts? all 3 of us could sync on this to get the right path forward quickly?

Reply:
Updating here - we've pushed ahead with the fact that a vendor should always show up in VendorRelationships every time, and we are only collecting the basic attributes in that section